[compiler] Add None type handling with sentinel value

bwasti opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #24 [WIP] Use PackedFunc to enable custom TVM operator registration from python
* #23 [benchmark] Add initial benchmark file
* #22 [compiler] Improve conversion from PyTorch IR to Relay
* #21 [testing] Improve testing to include models, image inputs and autotuned parameters
* **#17 [compiler] Add None type handling with sentinel value**
* #14 [compiler] Support operators with multiple outputs

bwasti committed:
[compiler] Add None type handling with sentinel value

wanchaol commented on torch_tvm/operators.cpp:
```diff
@@ -35,14 +35,35 @@ void registerSchedule(std::string name) {
+}
+
+uint64_t getNoneSentinel() {
+  return 0xe4fa3adecabcf036;
```
hmm this is kinda like a hack... what does this hex string means?

bwasti committed:
Update on "[compiler] Add None type handling with sentinel value"

[compiler] Add None type handling with sentinel value

gh-metadata: pytorch tvm 17 gh/bwasti/6/head

bwasti committed:
Update on "[compiler] Add None type handling with sentinel value"

[compiler] Add None type handling with sentinel value

gh-metadata: pytorch tvm 17 gh/bwasti/6/head

bwasti committed:
Update on "[compiler] Add None type handling with sentinel value"

[compiler] Add None type handling with sentinel value

gh-metadata: pytorch tvm 17 gh/bwasti/6/head

wanchaol reviewed (approved):
looks good

bwasti committed:
Update on "[compiler] Add None type handling with sentinel value"

[compiler] Add None type handling with sentinel value

gh-metadata: pytorch tvm 17 gh/bwasti/6/head
